docs/index.rst

```diff
@@ -244,6 +244,7 @@ each of which can be independently scaled to chase for the best end-to-end throu
    cpp_tutorials
    walkthrough_examples_acrossing_engines
    video_tutorials
+   gremlin_tutorials
```

Comment:
Can be place under cpp tutorials.

Reply:
updated. https://github.com/alibaba/GraphScope/pull/2369/commits/74f4323834c4e3f4a40a3a45e50c4095b1c8dd8c